coremltools/converters/mil/frontend/_utils.py

```diff
@@ -255,6 +255,32 @@ def solve_diagonal_einsum_one_step(parsed_vector, x):
     return tuple(parsed_vectors), vars
 
 
+def solve_sum_einsum(parsed_vectors, vars):
```

Comment:
It is not easy to follow this function,
@fukatani can you add some comment in the beginning of the function to explain what it trying to achieve?

Reply:
Thx!
I tried to write a comment. How about that?